src/stim/mem/simd_bits.test.cc

```diff
@@ -253,6 +317,23 @@ TEST_EACH_WORD_SIZE_W(simd_bits, left_shift_assignment, {
     }
     m0 <<= 488;
     ASSERT_TRUE(!m0.not_zero());
+    simd_bits<W> m1(448), m2(448);
```

Comment:
Separate the fuzz test out into a test with "fuzz" in the name, and put the body in a loop that repeats 5 times (increase it to 1000 while you write it but for detecting regressions 5 is more than enough IMO).